doc/modules/cross_validation.rst

```diff
@@ -223,10 +222,11 @@ learned using :math:`k - 1` folds, and the fold left out is used for test.
 Example of 2-fold cross-validation on a dataset with 4 samples::
 
   >>> import numpy as np
-  >>> from sklearn.cross_validation import KFold
+  >>> from sklearn.model_selection import KFold
 
-  >>> kf = KFold(4, n_folds=2)
-  >>> for train, test in kf:
+  >>> X = np.ones(4)
```

Comment:
Unlike tests, this is intended for users to read: do you think this kind of ad-hoc `X` can be confusing?

Alternatively we could have `X = ["a", "b", "c", "d"]`  maybe? (I'd suggest `[0, 1, 2, 3]` but then people might think that `split` returns the data itself rather than indices.)

Or am I overthinking this?